_layouts/integration.html

```diff
@@ -46,7 +58,7 @@ <h2>
 {{ content }}
 
 <section id="api-support">
```

Comment:
Change the id, and also places that refer to it above.

Just `machine-translation` I think.